docs/en_US/TrainingService/AMLMode.md

```diff
@@ -81,6 +81,13 @@ amlConfig:
     * optional key, default 1. Used to specify the max concurrency trial number on a GPU device.
 * useActiveGpu
     * optional key, default false. Used to specify whether to use a GPU if there is another process. By default, NNI will use the GPU only if there is no other active process in the GPU.
+* machineScheduler
```

Comment:
not suggest to expose this option in configuration. we can simply use the new scheduler for aml training service.